### PR TITLE
DPTS-15 Lookup capacity status api user password and build data job

### PR DESCRIPTION
**Pull request by jope1**
1. Parameterize DB script that creates the capacity_status_api role in the DoS DB.
2. Lookup password secret from /deployment secret store and inject into DB script at data job build time.
3. Configure build and deploy stages to use the TAG generated in the pipeline at initialisation stage (environment stage).
4. Refactor pipeline so that the data job is always rebuilt and the image stored in the ECR.
5. Refactor data job deployment process so that we pass in what version of the data job we want to deploy.

### CODE REVIEW

File: build/jenkins/Jenkinsfile.development [hunk 1]
```diff
@@ -11,6 +11,7 @@ pipeline {
   }
   environment {
     BUILD_DATE = sh(returnStdout: true, script: "date -u +'%Y-%m-%dT%H:%M:%S%z'").trim()
+    TAG = sh(returnStdout: true, script: "make project-get-tag BUILD_DATE=${BUILD_DATE}").trim()
```
Comment (stefaniuk) [inline build/jenkins/Jenkinsfile.development:14]: After integrating with the latest Make DevOps scripts this target will need to be renamed to `project-get-build-tag`

**Review by stefaniuk — DISMISSED** [review]
👍

**Review by dehe1 — DISMISSED** [review]
LGTM

**Review by timrickwood — APPROVED** [review]

File: deployment/stacks/service/overlays/dev/template/kustomization.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 bases:
   - ../../base
 
-patchesStrategicMerge:
-  - uec-dos-api-cs-proxy-ingress-customisation.yaml
+#patchesStrategicMerge:
```
Comment (timrickwood) [inline deployment/stacks/service/overlays/dev/template/kustomization.yaml:4]: Should we remove this altogether now we know it fixes the problem

**Review by timrickwood — CHANGES_REQUESTED** [review]

File: deployment/stacks/service/overlays/dev/template/kustomization.yaml (hunk 2 above)
Comment (jope1) [reply deployment/stacks/service/overlays/dev/template/kustomization.yaml:4]: Yes, will action

**Review by timrickwood — APPROVED** [review]

**Review by tamaragoldschmidt — APPROVED** [review]

**Pull request merged**